rero/rero-ebooks
# feat: ebook record dedup

How to test:
- git PR
- run_test.sh
- if this a new instance make sure to run bootstrap and setup
- run the following command to update existing records
dojson -l marcxml -i data/ebooks.xml do cantookmarc21 | invenio records create_else_update --vendor cantook


Signed-off-by: Aly Badr <aly.badr@rero.ch>
Co-authored-by: Gianni Pante <gianni.pante@rero.ch>
